wireapp/kalium
# chore: call manager

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

[AR-1327 - Request Call Configs when AVS needs it](https://wearezeta.atlassian.net/browse/AR-1327)
[AR-1326 - Receive Calling messages and pass them to AVS on Android](https://wearezeta.atlassian.net/browse/AR-1326)

### Issues

There was no way of creating a CallManager for different sessions.

### Solutions

This PR adds a Global Call Manager to handle the initialisation and the creation of a CallManager for AVS to handle calling.

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

Other parts will come to the epic

### Testing

Manually tested and tests will be done in another PR

1. branch kalium to this branch in android-reloaded project
2. delete and install the app
3. log in
4. from another account in web stage call the account logged in the app
5. in the logs you should see the callbacks being called with the log prefix: `CallManager - { ... }`
